### PR TITLE
chore(container): Correct webpack binding in container

### PR DESCRIPTION
**Pull request by bastilian**
This corrects the binding behaviour of webpack dev server when it is running within a container.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by akofink — COMMENTED** [review]
As discussed in Slack: this prevents the proxy from being able to access a local API (i.e. localhost:3000 for compliance-backend); however, the dns issue seems to be resolved.

**Comment by skateman** [comment]
I investigated further the issue with the DNS servers and if the host OS uses systemd-resolved, podman copies `/run/systemd/resolve/resolv.conf` for host networked containers, which in my case somehow only contains the DNS servers provided by my ISP. As this file is auto-generated, I tried to mess with the DNS priorities in my `NetworkManager` but that didn't really help me. 

If we're not using host networking for the proxy, the API is basically inaccessible as it is running in a separate pod and podman doesn't support [`host.docker.internal`](https://github.com/RedHatInsights/compliance-frontend/blob/master/config/spandx.config.js#L11) or anything similar like `host.podman.internal` and according to what I found, they are suggesting people to just use `--net=host` :confused:

**Comment by akofink** [comment]
I think we can close this one. As @skateman says, we were hitting issues with new Fedora 33 'features' :laughing:

**Comment by skateman** [comment]
I'm still struggling with the frontend not listening on `0.0.0.0` so I have to keep stashing stuff when git-pulling.

**Comment by akofink** [comment]
Ok, sure. @bastilian could we repurpose this PR to fix just the bind?

**Comment by bastilian** [comment]
@skateman The binding should now work out of the box.

**Review by skateman — APPROVED** [review]
The Seal of Approval

File: entrypoint.sh [hunk 1]
```diff
@@ -6,4 +6,6 @@ if [[ ! -d "/compliance/node_modules/@redhat-cloud-services" ]]; then
     cd /compliance; /usr/bin/npm install;
 fi
 
+export IN_DOCKER='true';
```
Comment (bastilian) [inline entrypoint.sh:9]: For some reason the Docker compose documentation is not right. A environment variable defined in the .env file doesn't seem to overwrite the one in the compose file[1].

[1] https://docs.docker.com/compose/environment-variables/

**Pull request merged**